tutorial: update of server URL and various outputs (#35)
mundialis/actinia_core: docs/installation.rst

        cd /tmp

>         wget https://grass.osgeo.org/grass76/source/snapshot/grass-7.6.git_src_snapshot_2019_07_27.tar.gz

Reviewer: Please update to grass78

>         wget https://grass.osgeo.org/grass76/source/snapshot/grass-7.6.git_src_snapshot_2019_07_27.tar.gz

Author: ```suggestion
        wget https://grass.osgeo.org/grass78/source/snapshot/grass-7.8.git_src_snapshot_latest.tar.gz
```

>         tar xzvf grass-7.6.git_src_snapshot_2019_07_27.tar.gz

Author: ```suggestion
        tar xzvf grass-7.8.git_src_snapshot_latest.tar.gz
```

>         mv grass-7.6.git_src_snapshot_2019_07_27 grass_master

Author: ```suggestion
        mv grass-7.8.git_src_snapshot_????_??_?? grass_master
```

        cd grass_master

>         export INTEL="-march=native -std=gnu99 -fexceptions -fstack-protector -m64"

Author: ```suggestion
        export INTEL="-std=gnu99 -fexceptions -fstack-protector -m64"
```

    ..

> 5. GRASS GIS 7.6 requires a Python2 environment for running, so we create one.

Reviewer: idem

> 5. GRASS GIS 7.6 requires a Python2 environment for running, so we create one.

Author: ```suggestion
5. GRASS GIS 7.8 requires a Python 3 environment for running, so we create one (not needed if only Python 3 is present on the machine).
```
